other/field-types/date.md

```diff
@@ -28,14 +28,16 @@ The date picker UI can be configured by using the `pikadayOptions` configuration
 }
 ```
 
-**Note: Apostrophe tries its best to convert any date picker format to the above mentioned** `YYYY-MM-DD` **friendly sorting format, but very advanced configurations may not work out of the box, so please keep that in mind.**
+{% hint style='info' %}
+Apostrophe tries its best to convert any date picker format to the above mentioned** `YYYY-MM-DD` friendly sorting format, but very advanced configurations may not work out of the box, so please keep that in mind.
 
-**If you do not set** `def: null` **or** `required: true`**, the date defaults to the current date.**
+If you do not set `def: null` or `required: true`, the date defaults to the current date.
+{% endhint %}
 
 ## Settings
 
 |  Property | Type   | Default | Description | 
 |---|---|---|---|
 | name | `string` | | Sets the name of the field in the database |
 | label | `string` | | Sets the label of the field that the user sees |
-| pikadayOptions | `float` |  | Allows configuration of the date format |
```

Comment:
Should be `object` for the type, no?

Reply:
A little bit of grandma's famous copypasta snuck in there...